server/sessionruntime/service.go

```
@@ -322,6 +456,26 @@ func (s *Service) resolveStore(ctx context.Context, sessionID string) (*session.
+	s.mu.Unlock()
+	if handle == nil {
+		return nil, nil
+	}
+	if err := waitForRuntimeHandleReady(ctx, handle); err != nil {
+		return nil, err
+	}
+	s.mu.Lock()
+	current := s.handles[trimmedSessionID]
+	s.mu.Unlock()
+	if current == nil || current != handle || current.activationErr != nil {
+		return nil, nil
```

golangci-lint (2.11.4)

[error] error is not nil (line 473) but it returns nil

(nilerr)